scripts/orchestrators/scanners.sh

```diff
@@ -42,6 +42,6 @@ run_armttk() {
     local bicep_file_path=$1
 
     az bicep build --file "${bicep_file_path}"
-    arm-ttk/Test-AzTemplate.sh "${bicep_file_path/.bicep/.json}"
+    arm-ttk/arm-ttk/Test-AzTemplate.sh "${bicep_file_path/.bicep/.json}"
```

Comment:
Is there a reason to have arm-ttk twice in the path?

Reply:
that's the correct path, you can find the successful execution logs here; https://dev.azure.com/enpolat/symphony_test_2/_build/results?buildId=573&view=results